internal/services/apimanagement/api_management_resource.go

```diff
@@ -320,14 +320,6 @@ func resourceApiManagementSchema() map[string]*pluginsdk.Schema {
 						Default:  false,
 					},
 
-					// TODO 4.0: change this from enable_* to *_enabled
-					"enable_frontend_ssl30": {
```

Comment:
was this supposed to be removed?

Reply:
Nope... I had to rebase my branch and this got removed by accident. Went over the diff with main and fixed this and a couple more issues.